business/src/main/java/org/mskcc/cbio/portal/service/ApiService.java

```diff
@@ -146,68 +149,25 @@ else if(type.equals("frequency"))
             
 		return results;
 	}
-        @Transactional
```

Comment:
@jjgao @JiaoJiao123 and @adamabeshouse all confirmed the `getMutationsCountsJSON` method can be removed.